javascript/features/gangs/test/mock_gang_database.js

```diff
@@ -23,12 +23,21 @@ class MockGangDatabase {
 
         switch (userId) {
             case MockGangDatabase.HKO_MEMBER_USER_ID:
-                gangInfo = { role: Gang.ROLE_MEMBER, useGangColor: false,
-                             gang: mockedGangInfo.hko };
+                gangInfo = { 
+                    role: Gang.ROLE_MEMBER, 
+                    useGangColor: false,
+                    gang: mockedGangInfo.hko,
+                    skinId: 80085
```

Comment:
Should we set a valid skin Id here and below? Just for the sake of having realistic-ish tests :)

Reply:
Totally agree!